.devcontainer/post_create_command.sh

```
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -e
+
+echo "🚀 Setting up PostHog development environment..."
+
+# Install uv (Python package manager)
+echo "📦 Installing uv..."
+curl -LsSf https://astral.sh/uv/install.sh | sh
+source ~/.local/bin/env
```

Shellcheck (0.10.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)